lib.php

```diff
@@ -161,9 +168,15 @@ public function embed($urls, $name, $width, $height, $options) {
             if (isset($options['subtitles'])) {
                 $tracks = array();
                 foreach ($options['subtitles'] as $label => $subtitlefileurl) {
-                    $tracks[] = array(
-                        'file' => $subtitlefileurl->out(),
-                        'label' => $label);
+                    if ($label == 'chapters' || $label == 'thumbnails') {
+                        $tracks[] = array(
+                            'file' => $subtitlefileurl->out(),
```

Comment:
Yes, that was my point.

By default the out method for a moodle_url has escaped set to true, so this needs to specify false to ensure we're using the unescaped version.